bin/set-centreon-master.sh

```diff
@@ -54,11 +54,11 @@ if [[ "$VIP_USED" == 1 ]] ; then
         fi
         echo "Unmounting VIP on ${CURRENT_MASTER_HOSTNAME}..."
         [[ "$DEBUG" ]] && set -x
-        VIP_SHUTDOWN_OUTPUT=$(sudo -u centreon ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${CURRENT_MASTER_HOSTNAME} -- sudo ifdown "${CENTRAL_VIP_IFNAME}\:1" 2>&1)
+        VIP_SHUTDOWN_OUTPUT=$(sudo -u centreon ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${CURRENT_MASTER_HOSTNAME} -- "sudo nmcli con mod ${CENTRAL_VIP_IFNAME} -ipv4.addresses ${CENTRAL_VIP_IPADDR}/${CENTRAL_VIP_CIDR_NETMASK} && sudo nmcli con up ${CENTRAL_VIP_IFNAME}" 2>&1)
```

Comment:
Is it what you wanted to replace "ifdown" with "con up" ?

Reply:
idem as previous